### PR TITLE
SONARKT-218 Update static analyzer docs

### PR DESCRIPTION
**Pull request by johann-beleites-sonarsource**

### CODE REVIEW

File: sonar-kotlin-plugin/src/main/resources/static/documentation.md [hunk 1]
```diff
@@ -12,6 +12,22 @@ key: kotlin
 
 You can discover and update Kotlin-specific [properties](/analysis/analysis-parameters/) in:  <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Kotlin](/#sonarqube-admin#/admin/settings?category=kotlin)**.
 
+## Kotlin Analysis and Bytecode
+If you are not using the Maven or Gradle Sonar plugins, it is strongly recommended to provide the paths of all dependency binaries used by
```
Comment (agigleux) [inline sonar-kotlin-plugin/src/main/resources/static/documentation.md:16]: If you are not using the `SonarScanner for Maven` or `SonarScanner for Gradle`, it is strongly ...

File: sonar-kotlin-plugin/src/main/resources/static/documentation.md [hunk 2]
```diff
@@ -12,6 +12,22 @@ key: kotlin
...
+the project in order to improve the analysis accuracy. You can provide these using the `sonar.java.libraries` property. This is a list of
```
Comment (agigleux) [inline sonar-kotlin-plugin/src/main/resources/static/documentation.md:17]: There is no typo here, it's really a Java property that must be used. There is no `sonar.kotlin.libraries` property.

File: sonar-kotlin-plugin/src/main/resources/static/documentation.md [hunk 3]
```diff
@@ -12,6 +12,22 @@ key: kotlin
 
 You can discover and update Kotlin-specific [properties](/analysis/analysis-parameters/) in:  <!-- sonarcloud -->Project <!-- /sonarcloud -->**[Administration > General Settings > Kotlin](/#sonarqube-admin#/admin/settings?category=kotlin)**.
 
+## Kotlin Analysis and Bytecode
+If you are not using the Maven or Gradle Sonar plugins, it is strongly recommended to provide the paths of all dependency binaries used by
+the project in order to improve the analysis accuracy. You can provide these using the `sonar.java.libraries` property. This is a list of 
+comma-separated paths to files with third-party libraries (JAR or Zip files) used by your project. Wildcards can be used: 
+`sonar.java.libraries=path/to/Library.jar,directory/**/*.jar`
+
+Note that if you use the Gradle or Maven Sonar plugins to scan your code, these plugins will usually auto-detect the value for the property.
```
Comment (agigleux) [inline sonar-kotlin-plugin/src/main/resources/static/documentation.md:21]: Note that if you use the "SonarScanner for Maven" or "SonarScanner for Gradle" to scan your code, these scanners will auto-detect the value for the property. Thus you don't need to provide them.

File: sonar-kotlin-plugin/src/main/resources/static/documentation.md [hunk 4]
```diff
@@ -12,6 +12,22 @@ key: kotlin
...
+
+## Specifying the Kotlin Source Code Version
+You can explicitly define which Kotlin version the analyzer should analyze your code based on. Provide the desired version in the format
+`X.Y` as value to the `sonar.kotlin.source.version` property (e.g. `1.7`).
+
+## Multi-Threaded Parsing
+As of August 2022, the SonarKotlin analyzer supports multi-threaded parsing. This is an experimental feature and is disabled by default.
```
Comment (agigleux) [inline sonar-kotlin-plugin/src/main/resources/static/documentation.md:28]: Note: "SonarKotlin": there is no such product existing :-)

As of August 2022, the Kotlin analyzer ...

**Review by agigleux — CHANGES_REQUESTED** [review]

**Review by agigleux — APPROVED** [review]

**Review by Wohops — APPROVED** [review]

**Pull request merged**